src/fiat/bank-authorities.md

```diff
@@ -191,14 +202,44 @@ curl -X POST "https://service.centrapay.com/api/bank-authorities/WRhAxxWpTKb5U7p
 }
 ```
 
-<span id="bank-authority-get"></span>
-## Get information about a bank account **EXPERIMENTAL**
+**Error Responses**
 
-{% endpoint GET https://service.centrapay.com/api/bank-accounts/${id} %}
+| Status |                           Code                            |                              Description                              |
+| :----- | :-------------------------------------------------------- | :-------------------------------------------------------------------- |
+| 403    | {% break _ BANK_ACCOUNT_ALREADY_VERIFIED %}               | The bank account is already verified                                  |
+| 403    | {% break _ BANK_ACCOUNT_VERIFICATION_ATTEMPTS_EXCEEDED %} | The number of times the user can verify the bank account has exceeded |
+| 403    | {% break _ BANK_ACCOUNT_MISMATCH %}                       | The top up and the bank account do not belong to the same account     |
```

Comment:
I am not sure what this error code should be called. Should it be BANK_AUTHORITY_ACCOUNT_MISMATCH or BANK_ACCOUNT_TOPUP_MISMATCH or BANK_ACCOUNT_MISMATCH?